tests/test_pointset.py

```diff
@@ -106,3 +106,157 @@ def test_filters_return_pointset(sphere):
     pointset = sphere.cast_to_pointset()
     clipped = pointset.clip()
     assert isinstance(clipped, pyvista.PointSet)
+
+
+@pytest.mark.parametrize("force_float,expected_data_type", [(False, np.int64), (True, np.float32)])
+def test_pointset_force_float(force_float, expected_data_type):
+    np_points = np.array([[1, 2, 3]], np.int64)
+    pset = pyvista.PointSet(np_points, force_float=force_float)
+
+    assert pset.points.dtype == expected_data_type
+
+
+def test_center_of_mass():
+    np_points = np.array([[0.0, 0.0, 1.0], [1.0, 0.0, 0.0]])
+    pset = pyvista.PointSet(np_points)
+    assert np.array_equal(pset.center_of_mass(), np.array([0.5, 0.0, 0.5]))
```

Comment:
This is looking great so far!

This comment applies to most, or possibly all, usages.  We should use `np.allclose` unless we know that the arrays will be unchanged.  In this example there may be numerical errors due to numerical precision.

There are a few failing tests for Windows due to this.

Reply:
```suggestion
    assert np.allclose(pset.center_of_mass(), [0.5, 0.0, 0.5])
```